src/components/Schematics/Schematic-Node.vue

```diff
@@ -116,6 +117,17 @@ export default {
         'font-size': `${size}px !important`,
         'line-height': `${size + 10}px !important`
       }
+    },
+    totalRuns() {
+      if (!this.mappedChildren) return
+
+      return Object.values(this.mappedChildren.state_counts).reduce(function(
+        a,
+        b
+      ) {
+        return a + b
+      },
+      0)
```

Comment:
I'm not sure this is necessary; don't we have this number in the `serialized_state.n_map_states` field on the task run?